.changeset/famous-comics-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/preview": patch
+---
+
+Fix an issue where "show code" was blocking loading in Storybook docs pages [CSS-1070]
```

Comment:
Is it worth noting what changes are being made here?
- Changes to the way Typekit is loaded (+ some more info about what these changes are)
- Icon now uses ref by default
- Removing some packages
- Excluding Variants decorator from rendering on the Docs pages
- anything else?